src/lib.rs

```diff
@@ -288,50 +286,49 @@ impl Preprocessor for Bibiography {
         NAME
     }
 
-    fn run(&self, ctx: &PreprocessorContext, mut book: Book) -> Result<Book, Error> {
+    fn run(&self, ctx: &PreprocessorContext, mut book: Book) -> Result<Book, anyhow::Error> {
         info!("Processor Name: {}", self.name());
 
-        if let Some(cfg) = ctx.config.get_preprocessor(self.name()) {
-            let bib_content = Bibiography::retrieve_bibliography_content(ctx, cfg);
-
-            if bib_content.is_err() {
+        let config = match Config::try_from(ctx.config.get_preprocessor(self.name())) {
```

Comment:
Not sure here. Should we generally only warn and skip on errors?

Reply:
Let's leave the warning for now and let's point to the README.md of the project. BTW, I forgot to mention in #9, do you want to add some info about the cited_only option in the README.md?